setup.py

```diff
@@ -1884,9 +1884,6 @@ def detect_tkinter(self):
         libs.append('tk'+ version)
         libs.append('tcl'+ version)
 
-        if HOST_PLATFORM in ['aix3', 'aix4']:
```

Comment:
Is not this needed on newer AIX versions?

Reply:
No. see earlier comment about my guess that this was fixed back when Linux Affinity was "added" to AIX.